### PR TITLE
Use system separator color in Separator component

### PR DESCRIPTION
**Pull request by vitalyster**
* border color is too distracting and does not look "native". Now we using system separator color to provide "native" experience :-)

### CODE REVIEW

File: src/utils/styles/themes.ts [hunk 1]
```diff
@@ -94,6 +103,12 @@ const themeColors: {
     dark_darker: 'rgb(90, 90, 90)'
   },
 
+  separator: {
+    light: PlatformColor(separatorColor, 'rgb(180, 180, 180)'),
```
Comment (xmflsct) [inline src/utils/styles/themes.ts:107]: Would just `PlatformColor(separatorColor)` be enough as it has `default`?

**Comment by xmflsct** [comment]
Thanks for the PR! Not knowing much about UIColor myself, it seems like it does support dark mode out of the box? If so, shall we apply the same logic to both dark modes of separator as well?

File: src/utils/styles/themes.ts (hunk 1 above)
Comment (vitalyster) [reply src/utils/styles/themes.ts:107]: PlatformColor default is used when it was unable to find provided system value. Default in Platform.select is used for non-iOS and non-Android platforms

**Comment by vitalyster** [comment]
Yes, it does. I don't know what two dark variants means there, that is why I did not touch dark modes

**Comment by xmflsct** [comment]
Cool, in that case we can just apply the same logic to both of the dark modes. :)

**Pull request merged**